src/Corax/Queries/MultiTermMatch.Erasure.cs

```diff
@@ -157,5 +158,11 @@ public static MultiTermMatch CreateEmpty(ByteStringContext context)
         {
             return new MultiTermMatch(new MultiTermMatch<EmptyTermProvider>(context, new EmptyTermProvider()), StaticFunctionCache<EmptyTermProvider>.FunctionTable);
         }
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public void Score(Span<long> matches, Span<float> scores) 
+        {
+            // We ignore. Nothing to do here. 
```

Comment:
Because it would be wrapped with `BoostingMatch` ?

Reply:
Yes, the idea is that `BoostingMatch` is the only one that assigns boosting values. We can totally externalize the behavior without having to mix boosting with anything else.